fe/fe-core/src/main/java/com/starrocks/sql/analyzer/CreateRoutineLoadAnalyzer.java

```diff
@@ -0,0 +1,33 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+package com.starrocks.sql.analyzer;
+
+import com.google.common.base.Strings;
+import com.starrocks.analysis.CreateRoutineLoadStmt;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReport;
+import com.starrocks.common.UserException;
+import com.starrocks.qe.ConnectContext;
+
+public class CreateRoutineLoadAnalyzer {
+    public static void analyze(CreateRoutineLoadStmt statement, ConnectContext context) {
+        String db = statement.getDBName();
+        if (Strings.isNullOrEmpty(db)) {
+            db = context.getDatabase();
+            if (Strings.isNullOrEmpty(db)) {
+                ErrorReport.reportSemanticException(ErrorCode.ERR_NO_DB_ERROR);
+            }
+        }
+        if (Strings.isNullOrEmpty(statement.getTableName())) {
+            ErrorReport.reportSemanticException(ErrorCode.ERR_BAD_TABLE_ERROR);
+        }
```

Comment:
you should check label name.
refer to CreateRoutineLoadStmt.analyze(Analyzer analyzer)

Reply:
[check label](https://github.com/StarRocks/starrocks/pull/9981/commits/33191d22863569624d23d28e271dea9133ddfe4c)